packages/sdk-browser/src/sdk-client.ts

```diff
@@ -140,13 +151,13 @@ export class TurnkeyBrowserSDK {
    */
   currentUserSession = async (): Promise<TurnkeyBrowserClient | undefined> => {
```

Comment:
@andrewkmin if we did this is your idea that we would return the `TurnkeyIframeClient` instead of the `TurnkeyBrowserClient` for read-write sessions?

Reply:
mm yeah that's tricky, i suppose it would have to be in that case

Reply:
I think we could abstract this in the react sdk
```
const { client } = useTurnkey()
```
we could make this `client` more intelligent by defaulting to creating a read/write session regardless of which client authenticates initially 
and the user would have the option of downgrading or setting a default auth level for the client in the config